helm/.env_jfrog_helm

```diff
@@ -0,0 +1,5 @@
+export NEWRELIC_LICENSE_KEY=change_me
```

Comment:
We should use `.env` extension instead. So `jfrog_helm.env` and this will help IDE/editor to detect file type correct.